src/components/live-kit-room/live-kit-room.scss

```
@@ -0,0 +1,8 @@
+@import "../../common/base";
+@import "../../common/icons";
+
+.button-container {
+  display: grid;
+  grid-template-columns: repeat(6, 1fr);
+  gap: 8px;
+}
```

Why do we need these styles?

```suggestion
:host {
  display: contents;
}

audio {
  display: none;
}
```